src/server/pkg/storage/fileset/fileset.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package fileset;
+option go_package = "github.com/pachyderm/pachyderm/src/server/pkg/storage/fileset";
+
+import "server/pkg/storage/fileset/index/index.proto";
+
+message Metadata {
```

Comment:
We have talked offline about this. It's not really Metadata. This object is in essence the fileset, not additional information.  `Layer`, `ODF`, `Op` `OpSet` `ChangeSet` `Patch` `Delta` are all better names.

Reply:
you could call it `INode`